pkgs/applications/misc/dstask/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "dstask";
+  version = "0.18";
+  rev = "d687a0f9aec634a808cfa0df4aecb06ab4f57732";
+
+  src = fetchFromGitHub {
+    owner = "naggie";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "16z5zlfj955pzsj0l58835slvpchdaq2vbyx2fjzi6y9xn1z2nd1";
+  };
+
+  vendorSha256 = null;
```

Comment:
Please add an inline comment explaining why this is needed. This may just be a link to the upstream author's explanation.

Reply:
Thanks! I've added a comment explaining this.